speaker_verification/model_evaluation.py

```diff
@@ -10,18 +10,18 @@
 from speaker_verification.audio import NUM_FBANKS, NUM_FRAMES, SAMPLE_RATE, read_mfcc, sample_from_mfcc
 from speaker_verification.rescnn_model import DeepSpeakerModel
 
-model_path = join(abspath(dirname(__file__)), "models", "ResCNN_triplet_training_checkpoint_265.h5")
+
+MODEL_PATH = join(abspath(dirname(__file__)), "models", "ResCNN_triplet_training_checkpoint_265.h5")
 
 def run_VCSK_Corpus_data(speaker_1, speaker_2, to_csv):
     np.random.seed(123)
     seed(123)
 
     model = DeepSpeakerModel()
-    model.rescnn.load_weights(model_path, by_name=True)
+    model.rescnn.load_weights(MODEL_PATH, by_name=True)
     dataset_path = (
-        "../../datasets/VCTK_Corpus_Fileshare/VCTK_Corpus/wav48_silence_trimmed"
+        "/home/aidan/dev/machine_learning/datasets/VCTK_Corpus_Fileshare/VCTK_Corpus/wav48_silence_trimmed/"
```

Comment:
Can you look into this value for `dataset_path` as it might need to be changed for the CI server?

Reply:
This code isn't run in the pytest tests, I've made a seperate `tests` directory with the test using `flac` files within the repository. This won't be an issue but probably should be refactored out of the module as it was primary goal was to create the `csv` files from the first iteration.